temporal-sdk/src/test/java/io/temporal/workflow/versionTests/GetVersionWorkflowReplaceGetVersionIdTest.java

```diff
@@ -83,7 +87,7 @@ public String execute(String taskQueue) {
         }
       }
       Workflow.sleep(1000); // forces new workflow task
-      return "test";
+      return hasReplayed;
```

Comment:
This PR has a mix of two styles. One is returning information about replaying through a global variable, another approach is returning in an `execute` return value. I think it's a good idea to unify the approaches.